dds/DCPS/RTPS/Spdp.cpp

```diff
@@ -1816,6 +1820,16 @@ Spdp::SpdpTransport::close()
   reactor_task_.stop();
 }
 
+void
+Spdp::SpdpTransport::bump_up_local_write_i()
+{
+  const TimeDuration quick_resend = outer_->config_->resend_period() * 0.1;
+  const TimeDuration min_resend = TimeDuration::from_msec(100);
```

Comment:
Move into condition.  Also, magic numbers.  At least define these as constants in the file or expose them as configuration values.